cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -17,12 +17,11 @@ metadata:
 spec:
   cephVersion:
     # The container image used to launch the Ceph daemon pods (mon, mgr, osd, mds, rgw).
-    # v12 is luminous, v13 is mimic, and v14 is nautilus.
-    # RECOMMENDATION: In production, use a specific version tag instead of the general v13 flag, which pulls the latest release and could result in different
+    # v12 is luminous, v13 is mimic, v14 is nautilus, and v15 is octopus.
```

Comment:
Should v15 really already be listed even if it is not supported yet?

Reply:
There won't be a `ceph/ceph:v15` tag until after it's released, so I'll remove it from this list for now